build.gradle

```diff
@@ -44,7 +44,7 @@ ext {
   versionCode = Integer.parseInt(VERSION_CODE)
   versionName = VERSION_NAME
 
-  minSdkVersion = 14
+  minSdkVersion = 26
```

Comment:
🤔 is this gon be okay

Reply:
Why was this change needed?

Reply:
i think it's fine? @alexkim205 wyt . Needed it for java time util